### PR TITLE
Test for SendIbcToken

### PR DESCRIPTION
**Pull request by phamminh0811**
## 1. Summary
Fixes #617

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

File: x/interchainstaking/keeper/receipt_test.go [hunk 1]
```diff
@@ -314,3 +317,58 @@ func (suite *KeeperTestSuite) TestReceiptStore() {
...
+		quicksilver.TransferKeeper.SetDenomTrace(
+			ctx,
+			transfertypes.DenomTrace{
+				Path:      fmt.Sprintf("%s/%s", portID, channelID),
+				BaseDenom: sdk.DefaultBondDenom,
+			},
+		)
+
+		// Try to send the ibc token
+		err = quicksilver.InterchainstakingKeeper.SendTokenIBC(ctx, sender, receiver, &zone, ibcAmount)
```
Comment (ThanhNhann) [inline x/interchainstaking/keeper/receipt_test.go:371]: I think we should check the change in the receiver's balance after `SendTokenIBC` success, and add one more test case for [this](https://github.com/quicksilver-zone/quicksilver/blob/a24dc7029618e046d66b2af2028925cca7ba6338/x/interchainstaking/keeper/receipt.go#L140)

Comment (phamminh0811) [reply x/interchainstaking/keeper/receipt_test.go:371]: There is no way to check the balance after testcase, refered to [TestSendTransfer](https://github.com/cosmos/ibc-go/blob/830982adfb4448573f8898bbd07a73d06bff640a/modules/apps/transfer/keeper/relay_test.go#L21) (ibc-go test)

Comment (ThanhNhann) [reply x/interchainstaking/keeper/receipt_test.go:371]: Ok, just need to add one more test case then we can merge it

Comment (phamminh0811) [reply x/interchainstaking/keeper/receipt_test.go:377]: @ThanhNhann added, please check

Comment (joe-bowman) [reply x/interchainstaking/keeper/receipt_test.go:376]: Is it worth checking the balance of the escrow module as a check?

```
amount := quicksilver.TransferKeeper.GetTotalEscrowForDenom(suite.chainA.GetContext(), coin.GetDenom())
suite.Require().Equal(expEscrowAmount, amount.Amount)
```

**Review by sontrinh16 — APPROVED** [review]

File: x/interchainstaking/keeper/receipt_test.go (hunk 1 above)
Comment (joe-bowman) [reply x/interchainstaking/keeper/receipt_test.go:376]: @phamminh0811 did the suggestion above not work?

Comment (phamminh0811) [reply x/interchainstaking/keeper/receipt_test.go:376]: there is no GetTotalEscrowForDenom function in transferkeeper

Comment (ThanhNhann) [reply x/interchainstaking/keeper/receipt_test.go:376]: Can you check [this](https://github.com/cosmos/ibc-go/blob/3d9f652d11cece3dbd9aa9c204d7ef6b43b111b3/modules/apps/transfer/keeper/keeper.go#L220) @phamminh0811

**Review by joe-bowman — APPROVED** [review]
lgtm

File: x/interchainstaking/keeper/receipt_test.go (hunk 1 above)
Comment (phamminh0811) [reply x/interchainstaking/keeper/receipt_test.go:376]: @ThanhNhann I think it is on ibc-go v8. So right now we don't have that function yet

**Pull request merged**